platform/security/encryption/platform-security-encryption-impl/src/main/java/ddf/security/encryption/impl/EncryptionServiceImpl.java

```diff
@@ -97,15 +98,12 @@ public synchronized String decrypt(String encryptedValue) {
   // @formatter:on
   @Override
   public String decryptValue(String wrappedEncryptedValue) {
-    String encryptedValue = unwrapEncryptedValue(wrappedEncryptedValue);
-    if (wrappedEncryptedValue == null) {
-      LOGGER.debug("A null password was provided.");
-      return null;
-    }
-    if (wrappedEncryptedValue.isEmpty()) {
-      LOGGER.debug("A blank password was provided in the configuration.");
-      return "";
+    if (StringUtils.isEmpty(wrappedEncryptedValue)) {
+      return wrappedEncryptedValue;
```

Comment:
❓  Should we add debug logging here?

Reply:
I thought about it and I didn't see the value. It the previous logging statements felt like noise to me.